### PR TITLE
Adds draft of WC creation workflow

### PR DESCRIPTION
**Pull request by swalner-vmware**
This PR adds the first pass of the workload cluster creation workflow.
The only "foundational" change is allowing handleValueChange to take objects (not just strings)

### CODE REVIEW

File: cli/cmd/plugin/ui/web/tanzu-ui/src/views/welcome/RolloverBanner/RolloverBanner.tsx [hunk 1]
```diff
@@ -32,8 +32,8 @@ const RolloverBanner = () => {
         {
             logo: TceLogo,
             icon: '',
-            backgroundImage: TceExperienceBg,
-            title: 'Community-supported experience',
+            // backgroundImage: TceExperienceBg,
```
Comment (miclettej) [inline cli/cmd/plugin/ui/web/tanzu-ui/src/views/welcome/RolloverBanner/RolloverBanner.tsx:35]: Hi Shimon, was it intentional to remove these background images?

File: cli/cmd/plugin/ui/web/tanzu-ui/src/views/workload-cluster/select-management-cluster.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.inputradio {
```
Comment (miclettej) [inline cli/cmd/plugin/ui/web/tanzu-ui/src/views/workload-cluster/select-management-cluster.scss:1]: nitpitck: rename to input-radio

File: cli/cmd/plugin/ui/web/tanzu-ui/src/views/workload-cluster/ClusterTopologyStep.tsx [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+}
+
+function InstanceTypeInList(instance: WorkerNodeInstanceType, register: any) {
+    return <>
+        <div className="text-white" cds-layout="col:1"><CdsIcon shape={instance.icon}></CdsIcon></div>
+        <input className="inputradio" cds-layout="col:1" type="radio" value={instance.id}
+               {...register("SELECTED_WORKER_NODE_INSTANCE_TYPE")} />
+        <div className="text-white" cds-layout="col:10">{instance.name} {instance.description}</div>
+    </>
+        ;
```
Comment (miclettej) [inline cli/cmd/plugin/ui/web/tanzu-ui/src/views/workload-cluster/ClusterTopologyStep.tsx:122]: nitpick, move ; to end of previous line

File: cli/cmd/plugin/ui/web/tanzu-ui/src/views/workload-cluster/ClusterTopologyStep.tsx [hunk 4]
```diff
@@ -0,0 +1,150 @@
+import React, { useContext } from 'react';
```
Comment (miclettej) [inline cli/cmd/plugin/ui/web/tanzu-ui/src/views/workload-cluster/ClusterTopologyStep.tsx:1]: when you get a chance, please organize these imports

File: cli/cmd/plugin/ui/web/tanzu-ui/src/views/welcome/RolloverBanner/RolloverBanner.tsx (hunk 1 above)
Comment (swalner-vmware) [reply cli/cmd/plugin/ui/web/tanzu-ui/src/views/welcome/RolloverBanner/RolloverBanner.tsx:35]: I will reenstate them. These were the files that weren't check in before. Good catch

File: cli/cmd/plugin/ui/web/tanzu-ui/src/views/workload-cluster/ClusterTopologyStep.tsx (hunk 3 above)
Comment (swalner-vmware) [reply cli/cmd/plugin/ui/web/tanzu-ui/src/views/workload-cluster/ClusterTopologyStep.tsx:122]: An issue of line length, enforced by eslint.

**Review by miclettej — APPROVED** [review]
Looks great Shimon

File: cli/cmd/plugin/ui/web/tanzu-ui/src/views/workload-cluster/ClusterTopologyStep.tsx [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+import { StepProps } from '../../shared/components/wizard/Wizard';
+import { WcStore } from '../../state-management/stores/Store.wc';
+
+interface ClusterTopologyStepFormInputs {
+    WORKLOAD_CLUSTER_NAME: string;
+    SELECTED_WORKER_NODE_INSTANCE_TYPE: string;
+}
+
+const clusterTopologyStepFormSchema = yup.object({
+    WORKLOAD_CLUSTER_NAME: yup.string().nullable().required('Please enter a name for your workload cluster').test('', 'Cluster name must contain only lower case letters and hyphen', value => value !== null && isValidClusterName(value)),
```
Comment (raymondz1) [inline cli/cmd/plugin/ui/web/tanzu-ui/src/views/workload-cluster/ClusterTopologyStep.tsx:28]: this line is super long. Is it possible to break it into multiple lines?

File: cli/cmd/plugin/ui/web/tanzu-ui/src/views/workload-cluster/SelectManagementCluster.tsx [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+const selectManagementClusterFormSchema = yup.object({
+    SELECTED_MANAGEMENT_CLUSTER_NAME: yup.string().nullable().required('Please select a management cluster')
+}).required();
+
+interface SelectManagementClusterFormInputs {
+    SELECTED_MANAGEMENT_CLUSTER_NAME: string;
+}
+
+const Description = styled.p`
+    padding: 20px;
```
Comment (raymondz1) [inline cli/cmd/plugin/ui/web/tanzu-ui/src/views/workload-cluster/SelectManagementCluster.tsx:27]: is it possible to use cdn-layout: p:sm to replace this padding?

File: cli/cmd/plugin/ui/web/tanzu-ui/src/views/workload-cluster/SelectManagementCluster.tsx [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+    line-height: 24px;
```
Comment (raymondz1) [inline cli/cmd/plugin/ui/web/tanzu-ui/src/views/workload-cluster/SelectManagementCluster.tsx:28]: there are also some line-height variables in typography.

File: cli/cmd/plugin/ui/web/tanzu-ui/src/views/workload-cluster/SelectManagementCluster.tsx [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+`;
+
+const SubTitle = styled.h3`
+    padding-left: 20px;
```
Comment (raymondz1) [inline cli/cmd/plugin/ui/web/tanzu-ui/src/views/workload-cluster/SelectManagementCluster.tsx:32]: use cdn-layout if possible

**Review by raymondz1 — APPROVED** [review]
a couple of suggestions. otherwise look good to me

**Pull request merged**